students/danwoj/Session13/test_timer.py

```diff
@@ -0,0 +1,10 @@
+import pytest, math
+
+from timer import Timer
+
+def test_timer():
+    with Timer() as t:
+        for i in range(90000):
+            i = i ** 20
+
+    assert math.isclose(t.interval, 0.05, rel_tol=0.00, abs_tol=0.1)
```

Comment:
I'm not sure you can count on the interval being consistent -- I don't think I"d test for that.

And yes, it's really hard to test this kind of thing.